client/src/components/SignIn/SignIn.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+
+import { loginUser } from './../../authentication';
+
+class Login extends Component {
```

Comment:
When you input something in the email field, the password field is empty and you try to sign in, it does not indicate an error that the password field is empty. If on the contrary, then there is a error. Should we prevent that?
Also, there are no messages from the server that let you know whether are sign in successfully or not.